agent/instrumentation/applicationinsights-web-2.3/build.gradle.kts

```diff
@@ -17,7 +17,7 @@ val otelVersion: String by project
 dependencies {
   compileOnly("com.microsoft.azure:applicationinsights-web:2.3.0")
 
-  testImplementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:$otelVersion")
+  testImplementation("io.opentelemetry.instrumentation:opentelemetry-instrumentation-annotations:$otelInstrumentationAlphaVersion")
```

Comment:
we had this for 2 years.

double check maven central: 

https://central.sonatype.com/artifact/io.opentelemetry.instrumentation/opentelemetry-instrumentation-annotations, it's using otelVersion.

Reply:
fixed